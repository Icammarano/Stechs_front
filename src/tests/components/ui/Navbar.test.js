import { shallow } from "enzyme";
import { Navbar } from "../../../components/ui/NavBar";

describe("Pruebas en <NavBar />", () => {
    test("Debe mostrarse correctamente", () => {
        const wrapper = shallow(<Navbar />);
        expect(wrapper).toMatchSnapshot();
    });
});
