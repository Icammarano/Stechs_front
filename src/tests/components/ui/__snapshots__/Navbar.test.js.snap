// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <NavBar /> Debe mostrarse correctamente 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark bg-dark"
>
  <Link
    className="navbar-brand p-1"
    to="/home"
  >
    Home
  </Link>
  <div
    className="navbar-collapse"
  >
    <div
      className="navbar-nav"
    >
      <NavLink
        activeClassName="active"
        className="nav-item nav-link"
        exact={true}
        to="/vendors"
      >
        Vendors
      </NavLink>
    </div>
  </div>
</nav>
`;
